src/index.html

```diff
@@ -1,10 +1,7 @@
 <!DOCTYPE html>
 <html>
 <head>
-    <script type="text/javascript" src="js/script.js" defer></script>
-    <script src="https://unpkg.com/react@16/umd/react.development.js" defer></script>
-    <script src="https://unpkg.com/react-dom@16/umd/react-dom.development.js" defer></script>
-    <script type="text/javascript" src="js/reactScript.js" defer></script>
+    <script type="text/javascript" src="js/bundle.js" defer></script>
```

Comment:
do you need the defer?

Reply:
Need defer so that page is loaded before script is. Otherwise some elements aren't loaded before the script tries to reference them